### PR TITLE
LitElement conversion

### PR DESCRIPTION
**Pull request by rgoessl**

### CODE REVIEW

**Comment by devpow112** [comment]
I was unable to ifnd the conversation but I remember there being issues with single quotes `'` in the lang js files. This is why a number of them use single quotes and then disable the warning eslint throws.
https://github.com/Brightspace/d2l-outcomes-overall-achievement/blob/master/lang/ar.js
https://github.com/Brightspace/d2l-outcomes-overall-achievement/blob/master/lang/.eslintrc.json
https://github.com/BrightspaceUI/core/blob/master/lang/ar.js
PRobably best to switch these files to use double quotes `"`.

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "lint:eslint": "eslint . --ext .js,.html",
     "format": "npm run -s format:eslint",
     "format:eslint": "npm run lint:eslint -- --fix",
-    "start": "es-dev-server --app-index demo/index.html --node-resolve --preserve-symlinks --dedupe --watch --open"
+    "start": "es-dev-server --app-index demo/index.html --node-resolve --dedupe --watch --open"
```
Comment (devpow112) [inline package.json]: I'd actually leave the `--preserve-symlinks` in. Shouldn't cause any issues but allows you o use `npm link` properly if we need to.

**Comment by devpow112** [comment]
I disabled `Verify Languages` as a required check so when this is good it's not blocked to merge.

**Comment by devpow112** [comment]
You'll want to add an .eslintrc override in the lang folder otherwise our config will hate the `"`. You can just copy this one https://github.com/Brightspace/d2l-outcomes-overall-achievement/blob/master/lang/.eslintrc.json

File: .eslintrc.json [hunk 2]
```diff
@@ -4,5 +4,8 @@
 	],
 	"globals": {
 		"Set": true
+	},
+	"rules": {
```
Comment (devpow112) [inline .eslintrc.json:8]: It's better to just add a config file for the subfolder `lang` that way it doesn't disable the rule for all the source just the lang folder. See previous comments for example file and placement

Comment (rgoessl) [reply .eslintrc.json:8]: True

**Comment by devpow112** [comment]
Might want to update the base eslint config file to use `lit-config` instead of polymer 3
https://github.com/Brightspace/d2l-outcomes-user-progress/blob/master/.eslintrc.json
Hopefully there won't be many problems

**Review by devpow112 — APPROVED** [review]
Looks good to me :tada:

**Pull request merged**